agent/stats/engine.go

```diff
@@ -228,59 +257,109 @@ func (engine *DockerStatsEngine) removeAll() {
 	}
 }
 
+func (engine *DockerStatsEngine) getStatsTaskContainer(task *apitask.Task, dockerContainerName string) {
+	var statsTaskContainer *StatsTask
+	if task.IsNetworkModeAWSVPC() {
+		numberOfContainers := len(task.Containers)
```

Comment:
should we exclude pause container from the calculation?

Reply:
Yes that's a great point, will do.